paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodForm.kt

```diff
@@ -0,0 +1,59 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.stripe.android.paymentsheet.forms.FormFieldValues
+import com.stripe.android.paymentsheet.forms.FormViewModel
+import com.stripe.android.paymentsheet.paymentdatacollection.FormFragmentArguments
+import com.stripe.android.ui.core.FormUI
+import com.stripe.android.ui.core.elements.FormElement
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.Flow
+
+@FlowPreview
+@Composable
+internal fun PaymentMethodForm(
+    args: FormFragmentArguments,
+    enabled: Boolean,
+    onFormFieldValuesChanged: (FormFieldValues?) -> Unit,
+    elementsFlow: Flow<List<FormElement>?>,
+    showCheckboxFlow: Flow<Boolean>,
+    modifier: Modifier = Modifier
+) {
```

Comment:
Together with a comment above, I wonder if we can get to a place where we simple provide the `PaymentMethodForm` with some arguments, and it’ll handle the rest itself. No need to pass in an `elementsFlow`, as this would be taken care of by the view model. Would love to discuss this!